README.md

```diff
@@ -198,7 +198,7 @@ The following parameters refer to components that are not internal to Codacy, bu
 | Parameter                        | Description                                                                                            | Default    |
 | -------------------------------- | ------------------------------------------------------------------------------------------------------ | ---------- |
 | `fluentdoperator.enable`         | Enable fluentd operator. It gathers logs from Codacy so that you can send it to our support if needed. | `false`    |
```

Comment:
Should the default be true?

Reply:
I agree with this, @paulopontesm @joaocosta-codacy what do you guys think?

Reply:
I would say yes to having it enabled by default. We do not want to put customers in a position where "we could help you if you had the logs". 
Do we know how many logs are exported/how big the logfiles are? My main concern is just going above a pvc limit for having verboseness enabled by default and the cluster getting clogged/slow/broken. But i do not possess that kind of information (yet) :) @paulopontesm ?

Reply:
Ok. 
We decided not to enable it by default because it was kind of intrusive since it creates a daemon set that attaches itself to the /var/log in all machines.
The idea was to have it disabled here and enabled in the `values-production.yaml`.

But I am confortable with having this enabled here. Just make sure you update the `values-production.yaml` so that it overrides values that are not default (remove the enable from there)